assets/javascripts/test_result.js

```diff
@@ -749,11 +749,36 @@ function renderInvestigationTab(response) {
         keyElement.style.verticalAlign = 'top';
         keyElement.appendChild(document.createTextNode(key));
 
+        var textLines = response[key].split('\n');
```

Comment:
I've just opened the investigation tab on a random failed job on my local instance and got this:

```
Uncaught TypeError: response[key].split is not a function
```

The problem is that one of the keys is rendered as an integer here. That's how `response` looks like in the job I have here:

```
diff_to_last_good: "-   "EXTERNAL_VIDEO_ENCODER_CMD" : "ffmpeg -hide_banner -nostats -r 24 -f image2pipe -vcodec ppm -i - -c:v libvpx-vp9 -crf 35 -b:v 1500k -cpu-used 0",↵+   "EXTERNAL_VIDEO_ENCODER_CMD" : "ffmpeg -y -hide_banner -nostats -r 24 -f image2pipe -vcodec ppm -i - -c:v libvpx-vp9 -crf 35 -b:v 1500k -cpu-used 0","
last_good: 1392
needles_diff_stat: ""
needles_log: "No needle changes recorded, test regression due to needles unlikely"
test_diff_stat: ""
test_log: "No test changes recorded, test regression unlikely"
```

So just check whether we've actually got a string here. If not just render it as-is.